Sources/Sentry/SentryClient.m

```diff
@@ -654,6 +654,10 @@ - (void)setSdk:(SentryEvent *)event
         if (self.options.stitchAsyncCode) {
             [integrations addObject:@"StitchAsyncCode"];
         }
+
+        if (self.options.enableCaptureFailedRequests) {
+            [integrations addObject:@"HTTPClientErrors"];
```

Comment:
Or `CaptureFailedRequests`?

Reply:
You gave a thumbs up, but approved it as well. Do you want me to rename this or keep this?

Reply:
Sorry. I think `CaptureFailedRequests` would be better as it matches the feature name in the options.